lua/lspconfig/server_configurations/lua_ls.lua

```diff
@@ -59,11 +59,11 @@ require'lspconfig'.lua_ls.setup {
             checkThirdParty = false,
             library = {
               vim.env.VIMRUNTIME
-              -- "${3rd}/luv/library"
+              -- "${3rd}/luv/library",
               -- "${3rd}/busted/library",
             }
             -- or pull in all of 'runtimepath'. NOTE: this is a lot slower
-            -- library = vim.api.nvim_get_runtime_file("", true)
+            -- library = vim.api.nvim_get_runtime_file("lua", true)
```

Comment:
this is not correct

Reply:
@glepnir Hey yep you are right. This turned out to be a misconfig on my side.

I had: (note the extra curly braces)
```lua
                {
                  workspace = {
                    checkThirdParty = false,
                    -- Make the server aware of Neovim runtime files
                    library = vim.api.nvim_get_runtime_file('', true),
                  },
                },
```

I remove those and now it works perfectly fine.
```lua
                workspace = {
                  checkThirdParty = false,
                  -- Make the server aware of Neovim runtime files
                  library = vim.api.nvim_get_runtime_file('', true),
                },
```

well... sometimes you have to go down the rabbit hole to find a couple of rogue curly braces I guess 😂. Feel free to close.